### PR TITLE
Register properties everywhere

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
cc @therealprof

**Comment by burrbull** [comment]
cc @Emilgardis 
I did not do this earlier because this is breaking change. But this should be done.

File: src/svd/registerinfo.rs [hunk 1]
```diff
@@ -414,7 +377,7 @@ mod tests {
                 <addressOffset>0x8</addressOffset>
                 <alternateGroup>alternate_group</alternateGroup>
                 <alternateRegister>alternate_register</alternateRegister>
-                <size>32</size>
+                <size>0x20</size>
```
Comment (Emilgardis) [inline src/svd/registerinfo.rs:380]: ?

Comment (burrbull) [reply src/svd/registerinfo.rs]: reverted

Comment (burrbull) [reply src/svd/registerinfo.rs]: cc @Emilgardis

**Pull request closed without merging**